google/cloud/channel/v1/cloud_channel_reports_client.h

```diff
@@ -165,11 +167,14 @@ class CloudChannelReportsServiceClient {
   /// [google.cloud.channel.v1.RunReportJobResponse.report_job]: @googleapis_reference_link{google/cloud/channel/v1/reports_service.proto#L169}
   ///
   // clang-format on
+  GOOGLE_CLOUD_CPP_DEPRECATED("This RPC is deprecated.")
   future<StatusOr<google::cloud::channel::v1::RunReportJobResponse>>
   RunReportJob(google::cloud::channel::v1::RunReportJobRequest const& request,
                Options opts = {});
 
   // clang-format off
+  /// @deprecated This RPC is deprecated.
+  ///
   ///
```

Comment:
The newlines in `FormatStartMethodComments()` are a bit off

Reply:
Done.